### DISCUSSION TITLE
Change layout to fill view

### DISCUSSION THREAD

**Discussion by mikekiwi72** [totaldebug/atomic-calendar-revive#494] [Support]
Hi,

I use the Revive calendar to have a good view from a distance (approx 2 meters) on a week-calendar on my 10" tablet. I upped the text-size to 150 and it works reasonably well. However, there is still a lot of blank space on the screen which I want to make use of.

So I'm looking for ways to adjust the layout to for instance "day-time-description" on one line so I can increase the text-sizes even more.

Or divide the screen into two columns so mon-thu are in 1 column and fri-sun on another. Problem then is that enlarging the font-size is causing the texts to overlap (already more or less the case at 150)>

Anyone who can give some directions on how to do this? Or am I asking for something that isn't possible (yet)? 

![Revive_1](https://user-images.githubusercontent.com/82221778/134865548-06ddf157-573d-4b65-b2f3-df1ddcb3a199.jpg)

![Revive_2](https://user-images.githubusercontent.com/82221778/134866554-a967c4d3-639d-4721-b3cd-f5a6ccea365f.jpg)

**Comment by marksie1988** [comment]
Hey thanks for the post, possibly something that could be done with card mod however it's not something ive tried so wouldn't be sure.

**Reply by mikekiwi72** [reply]
OK, will go into that direction to see if I can find something that helps me out.

Thanks!
